latest-coreth-commit.md

```diff
@@ -1,11 +1,12 @@
 # Latest Commits 
 
-The last coreth PR has been checked up until PR [#1125](https://github.com/ava-labs/coreth/pull/1125). Check the commit order in chronological order, rather than PRs in numeric order, as merge order is very frequently unaligned with PR open order. 
+The last coreth PR has been checked up until PR [#1135](https://github.com/ava-labs/coreth/pull/1125). Check the commit order in chronological order, rather than PRs in numeric order, as merge order is very frequently unaligned with PR open order. 
```

Comment:
```suggestion
The last coreth PR has been checked up until PR [#1135](https://github.com/ava-labs/coreth/pull/1135). Check the commit order in chronological order, rather than PRs in numeric order, as merge order is very frequently unaligned with PR open order. 
```